From 099423b412efea52a7318d14a4f18f9a2fd46bae Mon Sep 17 00:00:00 2001
From: Du Chenyang <chenyang.du@intel.com>
Date: Sat, 8 Jun 2013 11:25:10 +0800
Subject: [PDK MR2 AOSP] Replace inline with __inline__ in macro __BIONIC_FORTIFY_INLINE

BZ: 114804

umask multi-define error will occur in external/pcsc when try to
enable fortify. Replace inline with __inline__ to fix it from bionic.

Change-Id: I428babc9dd87d7ad424268e7e3120e2a075b9574
Signed-off-by: Du Chenyang <chenyang.du@intel.com>
---
 libc/include/sys/cdefs.h |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/libc/include/sys/cdefs.h b/libc/include/sys/cdefs.h
index 6d4abbd..01d58f8 100644
--- a/libc/include/sys/cdefs.h
+++ b/libc/include/sys/cdefs.h
@@ -524,7 +524,7 @@
 #endif
 
 #define __BIONIC_FORTIFY_INLINE \
-    extern inline \
+    extern __inline__ \
     __attribute__ ((always_inline)) \
     __attribute__ ((gnu_inline)) \
     __attribute__ ((artificial))
-- 
1.7.4.1

